Geometry_Engine/Modify/CleanPolyline.cs

```diff
@@ -36,11 +36,12 @@ public static partial class Modify
         /**** Public Methods                            ****/
         /***************************************************/
 
-        [Description("Returns a polyline defined by the points which result in deviations from a straight line only. For example, if a polyline has 3 points in a straight line, the middle point is removed as part of this cleaning process. This is designed for closed polylines only")]
+        [Description("Returns a polyline defined by the points which result in deviations from a straight line only. For example, if a polyline has 3 points in a straight line, the middle point is removed as part of this cleaning process if distance betwen point is less than distanceTolerance. This is designed for closed polylines only")]
         [Input("polyline", "The polyline you wish to clean by removing unnecessary points")]
-        [Input("tolerance", "The tolerance of the angle that defines a straight line. Default is set to the value defined by BH.oM.Geometry.Tolerance.Angle")]
+        [Input("angleTolerance", "The tolerance of the angle that defines a straight line. Default is set to the value defined by BH.oM.Geometry.Tolerance.Angle")]
+        [Input("distanceTolerance", "The distance between two point that should be removed. Default is set to the value 0.02")]
         [Output("polyline", "The cleaned polyline")]
-        public static Polyline CleanPolyline(this Polyline polyline, double angleTolerance = Tolerance.Angle, double distanceTolerance = Tolerance.Distance)
+        public static Polyline CleanPolyline(this Polyline polyline, double angleTolerance = Tolerance.Angle, double distanceTolerance = 0.02)
```

Comment:
I believe this should stay as before, `double distanceTolerance = Tolerance.Distance` too keep consistence across the whole repo. The default value can then be changed locally in further, discipline-specific methods.

Reply:
modified as discussed, this is not tolerance